core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/data/impl/ConfigServiceImpl.java

```diff
@@ -95,6 +102,26 @@ public int getThreadsPerCore() {
         return getProperty(PN_THREADS_PER_CORE, Integer.class).orElse(DEFAULT_THREADS_PER_CORE);
     }
 
+    @Override
+    public String getInternalLinksHost() {
```

Comment:
Can we discuss this? I have some difficulty understanding why one service must report config properties of _another_ service. Two other services, to be precise. Why these two services cannot report their own properties for themselves?...

The idea was, each service manages settings of its own (== setings that this concrete service needs for itself). But there will be some "orphaned" settings as well. The settings which are not "one's own" to any service. For these "orphaned" settings we create an "orphanage" which we name "ConfigService".

Also, I'm afraid that storing full path of an "...Impl" is not good. Why we bind to implementation instead of interface?... 
Again, I may be missing some point. Can we discuss?

Reply:
Okay here is a use case. I am the user of this project's API. I create a new service which follows the same interface as InternalLinkResolver and ExternalLinkResolver. This my service will have a setting, "My folder". The setting will be displayed in the Granite Console. But how this my service will read this setting?

Reply:
done